spec/requests/listings_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "Listings", type: :request do
+
+  describe "Listings" do
+    let(:user) { create(:user) }
+    let!(:listing_1) { create(:listing, user: user) }
+    let!(:listing_2) { create(:listing, user: user) }
+
+    subject { get api_v1_listings_path, headers: auth_headers, as: :json }
+
+    it 'Get all listings' do
```

Comment:
`it 'returns all listings'`

Reply:
create various contexts. 'When user is logged in', 'When user is not logged in'. Contexts usually come in pairs.